third_party/istio-latest/generate-manifests.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
It is not possible to move this script in `third_party/generate-manifest.sh` and share it with istio-latest, istio-head and istio-stable?

Reply:
I moved common code into `library.sh`. So what remains in each file is unique - ie. istio version and some override flags. So I'm not sure what the exact ask is and if you have a suggestion let me know.

Ideally `./hack/update-codegen.sh` could invoke these files. Consolidating these scripts into a single script (say at third_party) would be ideal but unfortunately with istio 1.8.4 the yaml ordering of `manifest generate` is not stable. So re-running it would produce diffs.